File: Java/1374_Generate_a_String_With_Characters_That_Have_Odd_Counts.java
class Solution {
    public String generateTheString(int n) {
        return n % 2 == 0 ? "a".repeat(n-1) + "b" : "a".repeat(n);
        /*
        StringBuilder result = new StringBuilder();
        if(n % 2 == 0){
            for(int i = 0; i < n-1; i++){
                result.append('a');
            }
            result.append('b');
        }
        else{
            for(int i = 0; i < n; i++){
                result.append('a');
            }
        }
        return result.toString();
        */
    }
}

File: CS/1374_Generate_a_String_With_Characters_That_Have_Odd_Counts.cs
public class Solution {
    public string GenerateTheString(int n) {
        StringBuilder sb = new StringBuilder();
        sb.Append('a', n-1);
        /*
        for(int i = 0; i < n-1; i++){
            sb.Append('a');
        }
        */
        sb.Append((n%2 == 0) ? 'b' : 'a');
        return sb.ToString();
    }
}
